### ISSUE TITLE
Error about test.py

### ISSUE THREAD

**Issue by lingbo666** [layumi/Person-reID-triplet-loss#19]
After python new_train.py,  I get some model(.pth).  
Then I try  python test.py --gpu_ids 0 --name ft_ResNet50 --test_dir /home/mist/lb666/Person-reID-triplet-loss/datasets/DukeMTMC-reID/pytorch --which_epoch 19. 
It will show error
![8567VX%EVQ()$6X6K UFMN](https://user-images.githubusercontent.com/56471702/81498308-b8e75580-92f6-11ea-8669-a5494ec7129b.png)
I am not sure if it's due to  I do not finish all train(I just train for 19 epoch), or something wrong about DUKE  and  prepare.py and test.py.
Thanks so much  in advance.!!!!!!

**Comment by layumi** [comment]
Duke do not have multi-query setting. You may consider to ignore and comment the multi-query code.

**Comment by lingbo666** [comment]
@layumi   Yeah, I find that DUKE  do not have  multi-query setting by reading  prepare.py in https://github.com/layumi/Person_reID_baseline_pytorch which is your another reid.
Now I try  prepare.py  and there is no  multi-query in pytorch, but still error. I think test.py may have something different and I will try to modify it.
Thanks for your two reids!
